packages/vuetify/src/components/VChip/VChip.ts

```diff
@@ -145,6 +145,9 @@ export default mixins(
         props: {
           right: true,
         },
+        attrs: {
```

Comment:
did you try `props: { size: 18 }`?

Reply:
Or use css to override the width/height/font-size on the `.v-chip__close` class if we want to keep the sass variable.

Reply:
@KaelWD I had tried fontSize and it didn't work but size did should be updated now :)